### ISSUE TITLE
new term request: resistant hypertension

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#673]
For new term requests, please provide the following information:

## Preferred term label
resistant hypertension

## Synonyms
drug resistant hypertension

## Textual definition
see: https://www.ncbi.nlm.nih.gov/pubmed/26935512

## Attribution
Tudor Oprea

**Comment by nicolevasilevsky** [comment]
**Email conversation below:**

From Chris: 

Let's add it!

---

On Wed, Apr 17, 2019 at 8:36 AM Tudor I Oprea wrote:
My “original use case” was a list from my chair, a nephrologist, about the “top 20 diseases” relevant to nephrology, and among them was “resistant hypertension”.  I looked it up, yes it’s a thing, it has 3000 entries!!! But no MeSH code, no ICD10 code. No MONDO or DO

https://www.ncbi.nlm.nih.gov/pubmed/26935512

as example

 ---

From: Chris Mungall:

We also have the tracker ticket I linked to, I like to have discussions there as other stakeholders can see.


But briefly, I don't see any issue with 1, just make a new term if required

 
I agree with 2-4, but this seems nothing to do with cross-products or consistency, just is it/when is it a good idea to have diseases named refractory-X given the context-dependent nature. But nevertheless these names seem to exist in the literature, people want them for a variety of purposes, e.g text mining, mapping from one disease resource to another (not sure what Tudor's original use case was), these things have to be balanced. One compromise is to add as needed but to place into subsets that contraindicate use for certain purposes (this is easy to do computationally). And we can always obsolete en-masse later, as we have done for rare-X.


---
On Wed, Apr 17, 2019 at 6:19 AM Peter Robinson wrote:

I think this is an important topic and one that we should not just discuss by email, since it seems we have a divergence of opinions.

Here are the reasons why I do not think this is a good idea

1. If your patient has refractory X-subtype but there are terms only for X-subtype and refractory X, which term do you choose?

2. Refractory (resistance) is a context dependent attribute and not an inherent characteristic of a disease. For instance, an incompetent physician might classify X as resistant-X if they are unable to find the treatment that cures X. Also, a patient with refractory-X might on Monday might have treatable-X on Tuesday if a new blockbuster medication is released.

3. Chronic diseases change all the time. For instance lupus and MS have flares where the treatments are not working, but then often cool down. Does the patient then cyclicly switch diagnoses from X to resistant-X?

4. In terms of counting diseases, it is necessary that we have a well defined system for deciding what is a disease and what is not a disease. If we wanted to, we could immediately double the number of rare diseases by defining therapy-resistant forms. I hope it is obvious that it is inconsistent to have resistant-X only for diseases where somebody makes a new-term request.

-Peter

 
---
From: Chris Mungall

It depends which sense we mean inconsistent. I don't think it's logically inconsistent. But any time you have compositional classes the hierarchy can be "inconsistent" in the sense of inconsistently populated, e.g. some diseases will have a refractory subtype named, others won't. I don't think this is a problem, we just add terms as we need them.

 ---

On Tue, Apr 16, 2019 at 5:55 PM Peter Robinson wrote:

Do you disagree that it is inconsistent? I think this is a modeling error. Possible hard to discuss via email.

---
From: Chris Mungall
 
We already have a DP for refractory (was using a MONDO qualifier term but switched to HP now)

I like the GO model: add terms if an annotator needs it, but don't try and populate the complete cross-product. If you need new terms later they will appear in the right place via reasoning.

We have various refractory X terms in MONDO at the moment. See: https://github.com/monarch-initiative/mondo/issues/671

---

On Tue, Apr 16, 2019 at 4:33 PM Peter Robinson wrote:

We definitely need to have a plan for how to deal with diseases like this.

The problem with this kind of disease definition is that it is overlapping with lots of other kinds of hypertension.

For instance, renovascular hypertension can be therapy resistant.

And so either we commit to making all of the cross products or we wind up with an inconsistent ontology.

I think that in the long run it will be much simpler to model things if we base our decisions on what is and what is not a

disease on the cross product of etiology and clinical management at a disease level.

We can easily add Clinical modifiers such as https://hpo.jax.org/app/browse/term/HP:0031375 to phenotypic signs, and this will make the

entire system more consistent.

I am hoping that once MONDO is in shape we can also stop using HPO terms that are on the borderline of disease/phenotype, this will also

make things more computable.

Assuming we are sufficiently funded, this will be a fun ride!

-Peter

 ---
From: Chris Mungall 
Sent: Tuesday, April 16, 2019 5:18 PM


Nicole, can you handle this one? Thanks!

---
On Tue, Apr 16, 2019 at 2:10 PM Tudor I Oprea wrote:

So “resistant hypertension” is definitely a thing

It’s (drug) resistant, meaning that current drugs don’t lower it

It’s not an exact synonym with “uncontrolled hypertension” since the latter may happen due to lack of compliance.

Could not find it in DO, nor MONDO (EBI) or the ICD10 dictionary

You do find it in PubMed…
